generator/transit_generator.cpp

```diff
@@ -282,7 +282,8 @@ void BuildTransit(string const & mwmDir, string const & countryId,
 
   auto const startOffset = w.Pos();
   Serializer<FileWriter> serializer(w);
-  header.Visit(serializer);
+  FixSizeNumberSerializer<FileWriter> numberSerializer(w);
+  header.Visit(numberSerializer);
```

Comment:
numberSerializer(header)
?

Reply:
FixSizeNumberSerializer and FixSizeNumberDeserializer have operator() only for numbers and enums. To do that I need to add a template operator() for classes which have Visit method. It makes FixSizeNumberSerializer and FixSizeNumberDeserializer more complicated.